src/app/oppdateringslogg/EnkeltOppdateringslogg.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { EnOppdateringslogg } from './OppdateringsloggContainer';
+import { Ingress, Systemtittel, Undertekst } from 'nav-frontend-typografi';
+import Tekstomrade from 'nav-frontend-tekstomrade';
+import styled from 'styled-components/macro';
+import Panel from 'nav-frontend-paneler';
+import Etikett from 'nav-frontend-etiketter';
+import Lesmerpanel from 'nav-frontend-lesmerpanel';
+import { formatterDatoTidMedMaanedsnavn } from '../../utils/date-utils';
+
+interface Props {
+    enOppdateringslogg: EnOppdateringslogg;
+    visMer: boolean;
+    setVisMer: (visMer: boolean) => void;
+}
+
+export enum OppdateringsloggType {
+    Beskjed = 'beskjed',
+    Oppdatering = 'oppdatering'
+}
+
+const StyledPanel = styled(Panel)`
+    margin: 1rem;
+`;
+
+const StyledEtikett = styled(Etikett)`
+    margin: 0rem;
+    float: right;
+`;
+
+const StyledDiv = styled.div`
+    text-align: center;
+    margin-top: 1rem;
+`;
+
+function Bilde({ src }: { src?: string }) {
+    if (!src) {
+        return null;
+    }
+    return (
+        <StyledDiv>
+            <img src={src} alt={''} width="100%" height="100%" />
+        </StyledDiv>
+    );
+}
+
+function OppdateringsloggEtikett({ type }: { type: OppdateringsloggType }) {
+    if (type === OppdateringsloggType.Beskjed) {
```

Comment:
Denne funker fint nå som vi har to typer, enten gjøre den mer generisk eller huske at vi må endre på det dersom det kommer fler typer. forslaget er en switch:

```
switch(type){
  case OppdateringsloggType.Beskjed:
    ....

default: 
return null
}
```